dom.go

```diff
@@ -750,6 +751,10 @@ func (d document) DocumentURI() string {
 	return d.Get("documentURI").String()
 }
 
+func (d document) ExecCommand(name string, showDefaultUI bool, valueArgument string) bool {
+	return d.Call("execCommand", name, showDefaultUI, valueArgument).Bool()
+}
```

Comment:
Will `""` ever be a valid argument? In a lot of other places, it is not, so we special-case `""` in the code to act as null. See for example `window.GetComputedStyle`. – I'd rather not see us use `*string` or `interface{}`.